reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/brownfield-byo-bastion.json

```diff
@@ -319,7 +319,7 @@
         "IpProtocol": "tcp",
         "FromPort": "10250",
         "ToPort": "10250",
-        "SourceSecurityGroupId": { "Fn::GetAtt": [ "MasterSG", "GroupId" ] }
+        "SourceSecurityGroupId": { "Fn::GetAtt": [ "NodeSG", "GroupId" ] }
```

Comment:
lemme know if the latest commit resolves that concern. I think thats what you meant